libpermian/cli/__init__.py

```diff
@@ -59,5 +59,13 @@ def main(*raw_args):
     if options.generate_event:
         print(event_spec)
         sys.exit(0)
+    if options.check_testruns:
+        crcList = get_caserunconfigurations(event_spec, options.settings, options.override)
+        print(f"""Would execute:
```

Comment:
I think this could be in a more machine readable format.
And if you have the lists of crcs/tcs/tps why not print them out instead of giving just the numbers.

Reply:
My original plan was not to provide any information and just use return code. I was also thinking about being able to combine the `--check-testruns` with `--generate-event` so that one could just execute the permian once and get both event specification string as well as information if anything for it would be tested. Thinking about it now, the print could be removed and the summary could be logged (INFO) as well as possible list of case run configurations that would be executed (DEBUG) which would also work as it would isolate standard output (possibly structured data) from these data.

Reply:
Just return code would be also fine, I just think that providing counts is an in-between solution that doesn't provide much of a benefit over return code. It all depends on the use case.

Reply:
What about now? I've used the logging, there's still no information about the test plans though (except the number of test plans) but I think this should be good enough for now.
My original thought with the numbers was to provide user something instead of plain return code when there's nothing to be executed so that it would be clear that no case run configuration was to be executed.

Reply:
Hmm, thinking about it now, it should be pretty easy to add information about the testcases and test plans to the debug log as well.